support-services/src/test/scala/com/gu/support/promotions/ServicesFixtures.scala

```diff
@@ -50,6 +51,12 @@ object ServicesFixtures {
     )
   )
   val guardianWeeklyWithCode = PromotionWithCode(NonGift.tenAnnual, guardianWeeklyAnnual)
+  val duplicate1 = promotion(validProductRatePlanIds, duplicatedPromoCode, discountBenefit)
+  val duplicate2 = promotion(validProductRatePlanIds, duplicatedPromoCode, freeTrial = freeTrialBenefit)
+  val duplicatedWithCode = List(
+    PromotionWithCode(duplicatedPromoCode, duplicate1),
+    PromotionWithCode(duplicatedPromoCode, duplicate1)
```

Comment:
Did you want to use `duplicate2` here?

Reply:
🤦  yes, good spot (although even at the time I thought it probably wasn't essential, just more realistic)

Reply:
actually `duplicatedWithCode` is not used anyway, I'll look into it

Reply:
Removed it.  it would only be needed to assert that the correct code was returned from findPromotion, but since the code can't be returned, there's no test for that.